package.json

```diff
@@ -11,7 +11,8 @@
   "types": "dist/src/index.d.ts",
   "files": [
     "dist",
-    "src"
+    "src",
+    "config"
```

Comment:
Not sure I have a strong opinion about this one.
It might be confusing that this will not be transpiled while `dist` is. Also, now the folder becomes a public API, meaning you will not be able to change it in the future.` index.js`-like is very helpful in these cases or even solutions like `import-path`.

What other libraries are doing in their build steps?

Reply:
Again, if we want users to use our preconfigured config file, then we should publish it, shouldn't we?
I went with Yoshi's convention of having config files in a config folder.

Reply:
`yoshi` doesn't have "build" steps, but we currently do because we are transpiling JSX code. so in the `yoshi` case, it is classic that the folder structure is the tool APIs.

Anyway, that's just something I had in mind for a while and don't have a best practice for

Reply:
I'm not sure I understand, are you talking about `src` or `config`?

Reply:
config

Reply:
I think for the consumer it won't really matter.
Also, I think the overhead of writing a file just for that, and publishing it (i.e. `import-path`-like) isn't worth it at this time.